samples/basic/full-authentication/dotnet/AspNetExtensions.cs

```diff
@@ -19,14 +19,14 @@
 using System.Net.Http;
 using System.Threading.Tasks;
```

Comment:
Feels like we need a readme as to what that Full Authentication sample is.

Reply:
There is one?